app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java

```diff
@@ -104,25 +110,26 @@ public void openCreateTicketFragment() {
     }
 
     private void setupRecyclerView() {
-        ticketsAdapter = new TicketsAdapter(getPresenter());
-
-        RecyclerView recyclerView = binding.ticketsRecyclerView;
-        recyclerView.setLayoutManager(new LinearLayoutManager(context));
-        recyclerView.setAdapter(ticketsAdapter);
-        recyclerView.setItemAnimator(new DefaultItemAnimator());
-        StickyRecyclerHeadersDecoration decoration = new StickyRecyclerHeadersDecoration(ticketsAdapter);
-        recyclerView.addItemDecoration(decoration);
-        recyclerView.addItemDecoration(new DividerItemDecoration(context, DividerItemDecoration.VERTICAL));
-
-        adapterDataObserver = new RecyclerView.AdapterDataObserver() {
-            @Override
-            public void onChanged() {
-                decoration.invalidateHeaders();
-            }
-        };
-
-        ViewUtils.setRecyclerViewScrollAwareFabBehaviour(recyclerView, binding.createTicketFab);
-
+        if (!initialized) {
```

Comment:
Can't be just check ticketAdapter == null

Reply:
Yes that would be better.

Reply:
Updated.